README.md

```diff
@@ -1,5 +1,128 @@
-# Custom Quiz Moodle Plugin
-__User stories__:
+# Contents
+
+0. [Introduction](/README.md#introduction)
+1. [Installation](/README.md#installation)
+2. [Adding Questions](/README.md#adding-questions)
+3. [Managing Questions](/README.md#managing-questions)
+4. [User Stories](/README.md#user-stories)
+5. [References](/README.md#references)
+
+# Introduction
+
+Abbreviations: MXML(Moodle XML)
+
+This plugin is used for creating custom quizzes using questions from question bank and letting the Evaluator choose the
+difficulty and topic for every question.
+The plugin also updates the `last_used` tag with the time the specific question was used,
+in order to avoid overusing some questions and ignoring others. The `last_used` tag can also be useful
+when reusing the questions in another Moodle instance
+
+# Installation
+
+### Installation From Downloaded zip file
+
+You can download the zip from:
+#### latest [(master branch)](https://github.com/systems-cs-pub-ro/quiz-manager-moodle/zipball/master)
+
+* Login as an admin and go to `Site administration > Plugins > Install plugins`
+* Upload the zip file. If you are prompted to add extra details, under `Plugin type` select `Question type (qtype)`
+* Go to `Show more > Rename the root directory` and type `quizmanager`
+* If your target directory is not writable, you will see a warning message. Proceed to
+[Manual installation](/README.md#manual-installation-from-zip-file)
+* Press `Install plugin from the ZIP file` button
+
+### Manual Installation From zip file
+
+If you are prompted with the message `Plugin type location /your/moodle/path/question/type is not writable`,
+you have to copy the files manually to the server file system, using a zip file or [Git](/README.md#manual-installation-using-git)
+
+* Download the [zip file](https://github.com/systems-cs-pub-ro/quiz-manager-moodle/zipball/master)
+* Unzip the file in the correct location: `/your/moodle/path/question/type`
+* Rename the directory to `quizmanager`
+
+### Manual Installation Using Git
+
+To install using git, type this command:
+```
+git clone https://github.com/systems-cs-pub-ro/quiz-manager-moodle.git /your/moodle/path/question/type/quizmanager
+```
+
+
+# Adding Questions
+
```

Comment:
Add a short summary: "Questions are added using the Moodle interface, after the plugin was installed."